pandas/tests/test_window.py

```diff
@@ -1138,6 +1149,28 @@ def alt(x):
 
             self._check_moment_func(f, alt, name='quantile', quantile=q)
 
+    def test_rolling_quantile_np_percentile(self):
+        # #9413
```

Comment:
add a 1-liner to both functions explaining what you are testing (e.g. the nature of the bug)